docs/upgrade/upgrade-to-enterprise/upgrade-guide-from-5.0-to-2022.1/upgrade-guide-from-5.0-to-2022.1-rpm.rst

```diff
@@ -0,0 +1,181 @@
+* CentOS 7 and 8
+
+Packages for Fedora are no longer provided.
+
+  
+Upgrade Procedure
+=================
+
+A ScyllaDB upgrade is a rolling procedure that  does not require a full cluster shutdown. For each of the nodes in the cluster, serially (i.e. one at a time), you will:
+
+* Check the cluster schema
+* Drain the node and backup the data
+* Backup the configuration file
+* Stop ScyllaDB
+* Download and install new ScyllaDB packages
+* Start ScyllaDB
+* Validate that the upgrade was successful
+
+Apply the following procedure **serially** on each node. Do not move to the next node before validating that the node is up and running the new version.
+
+**During** the rolling upgrade, it is highly recommended:
+
+* Not to use new 2022.1 features.
+* Not to run administration functions, like repairs, refresh, rebuild or add or remove nodes. See :doc:`here </operating-scylla/manager/2.1/sctool>` for suspending Scylla Manager scheduled or running repairs.
+* Not to apply schema changes.
+
+Upgrade Steps
+=============
+
+Check the cluster schema
+--------------------------
+
+Make sure that all nodes have the schema synched before the upgrade. The upgrade will fail if there is a schema disagreement between nodes.
+
+.. code:: sh
+
+       nodetool describecluster
+
+Drain node and backup the data
+------------------------------
+Before any major procedure, like an upgrade, it is recommended to backup all the data to an external device. In ScyllaDB, backup is done using the ``nodetool snapshot`` command. For **each** node in the cluster, run the following command:
+
+.. code:: sh
+
+   nodetool drain
+   nodetool snapshot
+
+Take note of the directory name that nodetool gives you, and copy all the directories having this name under ``/var/lib/scylla`` to a backup device.
+
+When the upgrade is completed on all nodes, the snapshot should be removed with the ``nodetool clearsnapshot -t <snapshot>`` command, or you risk running out of space.
+
+Backup the configuration files
+--------------------------------
+.. code:: sh
+
+   for conf in $( rpm -qc $(rpm -qa | grep scylla) | grep -v contains ) /etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; do sudo cp -v $conf $conf.backup-5.0; done
+
+Stop ScyllaDB
+--------------
+.. code:: sh
+
+   sudo systemctl stop scylla-server
+
+Download and install the new release
+------------------------------------
+Before upgrading, check what version you are running now using ``rpm -qa scylla\*server``. You should use the same version in case you want to :ref:`rollback <upgrade-5.0-2022.1-rpm-rollback-procedure>` the upgrade. If you are not running a 5.0.x version, stop right here! This guide only covers 5.0.x to 2022.1.y upgrades.
+
+To upgrade:
+
+#. Update the `ScyllaDB RPM Enterprise repo <http://www.scylladb.com/enterprise-download/centos_rpm/>`_  to **2022.1**.
+#. Install:
+
+    .. code:: sh
+
+       sudo yum clean all
+       sudo rm -rf /var/cache/yum
+       sudo yum remove scylla\*
```

Comment:
@annastuchlik / @tzach in case the base version (5.0 or earlier) was installed using a scylla image (e.g. AMI).
There is another step that should be taken here:

For  Ubuntu20:
'''
sudo apt-get install scylla-enterprise-machine-image -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"
'''

For CentOS (old AMI that was upgraded to 5.0):
'''
sudo yum install scylla-enterprise-machine-image
'''

Reply:
@soyacz please review as well

Reply:
I'm not sure if we need to add `-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold"` - this setting makes apt not to update modified config files (without prompt). But anyway we restore config files from backup later.
We use this part in our tests because this is the best way to answer prompts automatically.

Reply:
Edited my comment.

Reply:
@roydahan I've updated the installation step. Could you re-review?